packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -35,6 +36,7 @@ import { isWpComProductRenewal } from './is-wpcom-product-renewal';
 import { joinClasses } from './join-classes';
 import { getPartnerCoupon } from './partner-coupon';
 import IonosLogo from './partner-logo-ionos';
+import type { SiteDetails } from '@automattic/data-stores';
```

Comment:
This won't (correctly) work because the `@automattic/wpcom-checkout` package currently doesn't have a `@automattic/data-stores` listed as a dependency. In order to make this import correct, we'd need to add that dependency (at least just for TypeScript... I'm not sure if there's a special way to do that or not). That will add a lot of overhead to the package.

A simpler solution might be to duplicate the `SiteDetails` type right here (you don't even need to duplicate the whole thing, just the parts you actually use below).